test2code/src/main/kotlin/com/epam/drill/test2code/coverage/RetentionQueue.kt

```diff
@@ -28,22 +28,21 @@ interface RetentionQueue {
 
 class InMemoryRetentionQueue(
     private var queue: Queue<ByteArray> = ConcurrentLinkedQueue(),
-    private val totalSizeByteLimit: BigInteger,
-    //TODO PR: check additional  Unit param. Investigate Parsing Mb to Unit.MegaBytes e.t.c
+    private val totalSizeByteLimit: BigDecimal
```

Comment:
Reason for using decimal. Number of bytes is an integer value, no need for floating point

Reply:
done